website/static/js/filepage/revisions.js

```diff
@@ -2,6 +2,8 @@ var m = require('mithril');
 var $ = require('jquery');
 var $osf = require('js/osfHelpers');
 var waterbutler = require('js/waterbutler');
+var bootbox = require('bootbox');
```

Comment:
Group imports according to our style guidelines: https://cosdev.readthedocs.org/en/latest/style_guides/javascript.html#commonjs-modules